samples/browser/src/main/java/org/mozilla/samples/browser/BaseBrowserFragment.kt

```diff
@@ -209,13 +196,28 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler {
         // Observe the lifecycle for supported features
         lifecycle.addObservers(
             scrollFeature,
-            menuUpdaterFeature,
             secureWindowFeature
         )
 
         return layout
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        consumeFlow(components.store) { flow ->
+            flow.mapNotNull { state -> state.findCustomTabOrSelectedTab(sessionId) }
+                .ifAnyChanged { tab ->
```

Comment:
Should we add a test to verify that we are observing the store when `onViewCreated`?

Reply:
This is Sample Browser.

Reply:
We're not unit testing / just UI testing our Sample apps. I don't think we want to change that?

Reply:
(But we do want more UI tests in sample browser for sure) :)